Assets/Scripts/CameraLookAt.cs

```
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraLookAt : MonoBehaviour
+{
+    private FocalPoint _focalPoint;
+
+    private void Start()
+    {
+        _focalPoint = FindObjectOfType<FocalPoint>();
```

Fail fast пжалста, вместо Null-refов которые можно проигнорить